Mathlib/Analysis/Calculus/EDeriv/Basic.lean

```diff
@@ -0,0 +1,176 @@
+/-
+Copyright (c) 2025 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov, Sam Lindauer
+-/
+import Mathlib.Analysis.NormedSpace.Alternating.Uncurry.Fin
+import Mathlib.Analysis.Calculus.FDeriv.Symmetric
+import Mathlib.Analysis.Calculus.FDeriv.CompCLM
+
+/-!
+# Exterior derivative of a differential form
+
+In this file we define the exterior derivative of a differential form.
+Under certain smoothness assumptions, we prove that this operation is linear in the form
+and the second exterior derivative of a form is zero.
+-/
+
+open Filter ContinuousAlternatingMap Set
+open scoped Topology
+
+variable {𝕜 E F : Type*}
+  [NontriviallyNormedField 𝕜]
+  [NormedAddCommGroup E] [NormedSpace 𝕜 E]
+  [NormedAddCommGroup F] [NormedSpace 𝕜 F]
+  {n m k : ℕ} {r : WithTop ℕ∞}
+  {ω ω₁ ω₂ : E → E [⋀^Fin n]→L[𝕜] F} {s t : Set E} {x : E}
```

Comment:
Perhaps this should be a follow-up: this is crying for the use of notation...

Reply:
I failed to come up with different notation for differential forms on normed spaces / manifolds, unbundled or bundled C^k.